### PR TITLE
[Feature] FindPassword and ChangePassword integration

### PR DESCRIPTION
**Pull request by seonghyeonkimm**
## Description
- [x] FindPassword
* assignee: @devethan, @seonghyeonkimm
- [x] Show friends
* assignee: @irrationnelle, @seonghyeonkimm

## Related Issues
- #98 

## Tests
I added the following tests:
- Run `yarn test FindPw.test.tsx` and `yarn test ChangePw.test.tsx`
- Added tests to check whether the alert is called after FindPw and ChangePw API are processed correctly.

## Checklist
Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk-mobile/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by seonghyeonkimm** [comment]
@irrationnelle CLA sign 해주실 수 있을까요?

**Comment by seonghyeonkimm** [comment]
I modified a little logic and test code based on you built before. Please review and leave comments!
@devethan @irrationnelle

**Comment by hyochan** [comment]
@seonghyeonkimm Please request a review if you are ready!

**Comment by seonghyeonkimm** [comment]
Ah, I didn't know that I should request reviews. Could you review this PR?  @hyochan

File: src/components/screen/ChangePw.tsx [hunk 1]
```diff
@@ -38,26 +40,44 @@ function ChangePw(props: Props): ReactElement {
...
+
+  const navigateBack = (): void => navigation.goBack();
 
   const changePassword = async (): Promise<void> => {
-    if (newPw === confirmPw) {
-      // TODO change password api call
-      Keyboard.dismiss();
-      Alert.alert('', 'Password changed.', [
+    if (newPw !== confirmPw) {
+      Alert.alert(getString('PASSWORD_MUST_MATCH'));
```
Comment (hyochan) [inline src/components/screen/ChangePw.tsx]: How about `Alert.alert(getString('ERROR'), getString('....)`?
The long title for Alert dialog is quite ugly.

File: src/components/screen/ChangePw.tsx [hunk 2]
```diff
@@ -38,26 +40,44 @@ function ChangePw(props: Props): ReactElement {
...
+        Keyboard.dismiss();
+        Alert.alert('', getString('PASSWORD_IS_CHANGED'), [
+          {
+            text: getString('OK'),
+            onPress: navigateBack,
+          },
+        ]);
+      }
+    } catch (e) {
+      Alert.alert('', getString('CHANGE_PASSWORD_IS_FAILED'), [
```
Comment (hyochan) [inline src/components/screen/ChangePw.tsx]: `CHANGE_PASSWORD_IS_FAILED` => `CHANGE_PASSWORD_HAS_FAILED`
or `CHANGE_PASSWORD_FAILED`

**Comment by seonghyeonkimm** [comment]
@hyochan I just applied for your reviews. (However, I could not test on stage server because I suddenly have build error like below. I didn't solve the error until now)

```
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening HackaTalk.xcworkspace. Run CLI with --verbose flag for more details.
```

**Comment by hyochan** [comment]
> @hyochan I just applied for your reviews. (However, I could not test on stage server because I suddenly have build error like below. I didn't solve the error until now)
> 
> ```
> error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening HackaTalk.xcworkspace. Run CLI with --verbose flag for more details.
> ```

I am suspecting it's because of the `virgile sdk`. Could you try to see [our troubleshoot](https://github.com/dooboolab/hackatalk-mobile#troubleshoot)?

**Comment by seonghyeonkimm** [comment]
Hmm, It still doesn't work even if I added code to node_modules described in the link you shared. 😭

**Comment by hyochan** [comment]
Have you tried pod repo update && pod install?

File: src/components/screen/ChangePw.tsx [hunk 3]
```diff
@@ -38,26 +40,44 @@ function ChangePw(props: Props): ReactElement {
   const [currentPw, setCurrentPw] = useState('');
   const [newPw, setNewPw] = useState('');
   const [confirmPw, setConfirmPw] = useState('');
-  const close = (): void => {
-    navigation.goBack();
-  };
+  const [mutationChangePassword] = useMutation<{ changeEmailPassword: boolean }, MutationChangePasswordInput>(
```
Comment (hyochan) [inline src/components/screen/ChangePw.tsx]: `mutationChangePassword` ==> `changePassword` would be better naming

Comment (seonghyeonkimm) [reply src/components/screen/ChangePw.tsx]: I changed the name of the variables. Thanks!

**Comment by hyochan** [comment]
@seonghyeonkimm Test is failing. Please update jest snapshot.

**Comment by seonghyeonkimm** [comment]
Sorry for the late reply! I did updated the snapshots! @hyochan

**Pull request merged**